node-graph/gcore/src/vector/generator_nodes.rs

```diff
@@ -52,24 +52,16 @@ impl CornerRadius for f64 {
 impl CornerRadius for [f64; 4] {
 	fn generate(self, size: DVec2, clamped: bool) -> super::VectorData {
 		let clamped_radius = if clamped {
-			// no corner's radius can be larger than the smallest side
-			let mut radius = self.map(|x| x.clamp(0., size.x.min(size.y).max(0.)));
-			// find the corner with the largest radius
-			let (mut max, mut index) = (0., 0);
-			(0..4).for_each(|i| {
-				if radius[i] > max {
-					(max, index) = (radius[i], i);
+			// https://drafts.csswg.org/css-backgrounds/#corner-overlap
+			let mut f: f64 = 1.;
+			for i in 0..4 {
+				let l = if i % 2 == 0 { size.x } else { size.y };
+				let s = self[i] + self[(i + 1) % 4];
+				if l < s {
+					f = f.min(l / s);
```

Comment:
Please use descriptive variable names, not letters. (Except `i` which is the standard iterator variable.)

Reply:
Sure!

Reply:
Cool, I've renamed that to `adjacent_corner_radius_sum` now.